### ISSUE TITLE
Hi, discuss

### ISSUE THREAD

**Issue by jerviscui** [mvali/Mapping-packages-Benchmark#1]
How about show the result in README.md.

And, can you tell me which is your choice?

**Comment by mvali** [comment]
Good point. I have listed the results. I'm using Automapper  and sometimes the manual map but these results may vary depending on system and compiler used. For some people Mapster had good results.

**Comment by jerviscui** [comment]
Em, the result is a bit strange?

AutoMapper time is 1/9 of ManualMap, so AutoMapper is the fastest?

**Comment by jerviscui** [comment]
Thank you for your efforts, this is my result.

BenchmarkDotNet=v0.12.1, OS=Windows 10.0.19042
Intel Xeon CPU E5-2667 v4 3.20GHz, 1 CPU, 16 logical and 8 physical cores
.NET Core SDK=5.0.104
  [Host]     : .NET Core 5.0.4 (CoreCLR 5.0.421.11614, CoreFX 5.0.421.11614), X64 RyuJIT  [AttachedDebugger]
  DefaultJob : .NET Core 5.0.4 (CoreCLR 5.0.421.11614, CoreFX 5.0.421.11614), X64 RyuJIT


|                    Method |      Mean |    Error |    StdDev |    Median |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|-------------------------- |----------:|---------:|----------:|----------:|-------:|------:|------:|----------:|
|                 ManualMap |  93.79 ns | 1.875 ns |  3.028 ns |  93.37 ns | 0.0275 |     - |     - |     360 B |
| MapsterAdaptWithoutConfig | 123.22 ns | 2.459 ns |  6.079 ns | 121.34 ns | 0.0267 |     - |     - |     352 B |
|    MapsterAdaptWithConfig | 153.10 ns | 3.053 ns |  5.427 ns | 152.88 ns | 0.0267 |     - |     - |     352 B |
|        MapsterAdaptToType | 175.60 ns | 3.476 ns |  4.003 ns | 174.91 ns | 0.0298 |     - |     - |     392 B |
|                AutoMapper | 249.12 ns | 4.982 ns | 11.345 ns | 244.68 ns | 0.0272 |     - |     - |     360 B |
|             ExpressMapper | 346.23 ns | 6.820 ns | 14.235 ns | 346.29 ns | 0.0396 |     - |     - |     520 B |

Run time: 00:03:44 (224.13 sec), executed benchmarks: 6

**Comment by jerviscui** [comment]
I'm learning Mapster too.

I see Mapster create a complier FEC and has better performance.
I tried the official Benchmark, [https://github.com/jerviscui/Mapster/blob/ref/src/Benchmark/README.md](url)

**Comment by mvali** [comment]
I knew that Mapster could had better result then mine and that was one of the reasons I did not added the benchmark. So people can download and try for themselves. The only required nuget packages are 3 for presented mappers and the benchmark so easy to integrate further testing. If you allow me I can list your benchmark as well on repository description.

**Comment by jerviscui** [comment]
Sure, of course you can list the result.
I just to run the results. The code is https://github.com/MapsterMapper/Mapster/tree/master/src/Benchmark
